src/components/EditSections/EditUserInfo/constants.js

```diff
@@ -0,0 +1,9 @@
+export const USER_TYPES = {
+  staff: 'staff',
+  patron: 'patron',
+  shadow: 'shadow',
+};
+
+export const checkIfConsortiumEnabled = stripes => {
```

Comment:
You probably forgot to remove this func from constants when moved it :)

Reply:
you are right. 1 sec